### PR TITLE
Add facewear randomization (CBA)

### LINKED ISSUE #24: Add facewear randomization
Appears to be broken with both `allowedFacewear[]` config and direct array fill with `BIS_fnc_unitHeadgear`.

~~http://feedback.arma3.com/view.php?id=26146~~
https://feedback.bistudio.com/T84513

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Replace #112 when https://github.com/CBATeam/CBA_A3/pull/1201 releases
- Close #24 
- Set default headgear properly - for when randomization is disabled.

**Merge after https://github.com/CBATeam/CBA_A3/pull/1201 is released!**

### CODE REVIEW

File: addons/units/CfgVehicles_PublicUnits.hpp [hunk 1]
```diff
@@ -274,6 +334,10 @@ class CLASS(Unit_I_PilotHeli): CLASS(Unit_Combat_RS_BS_GP_BB) {
             speechPlural[] = {"veh_infantry_pilot_p"};
         };
     };
+
+    CBA_facewearList[] = {
+        "", 1
+    };
```
Comment (Kresky) [inline addons/units/CfgVehicles_PublicUnits.hpp]: If the facewearList only has one entry anyway, isn't it a bit redundant to add it?

Comment (jonpas) [reply addons/units/CfgVehicles_PublicUnits.hpp]: No, read documentation: https://github.com/CBATeam/CBA_A3/blob/ac776664cbc160744f58c539a065ffefe3aa4cb6/addons/common/fnc_randomizeFacewear.sqf#L9

Comment (Kresky) [reply addons/units/CfgVehicles_PublicUnits.hpp]: Ah, didn't see the doc yet.
Thought if you just had one facewear applied in linkedItems, you wouldn't need the facewearList, similar to the headgear.

All looks good to me then.

Comment (jonpas) [reply addons/units/CfgVehicles_PublicUnits.hpp]: `linkedItems` is just the default. If you inherit a unit with randomization you might still have randomization, this disables it 100%.

**Pull request merged**